packages/gatsby/playground-sandbox/pages/index.js

```diff
@@ -1,8 +1,7 @@
 import Head                from 'next/head';
 import {useRef, useEffect} from 'react';
 
-// TODO: replace with `https://yarnpkg.com`
-const WEBSITE_URL = `https://deploy-preview-1193--yarn2.netlify.com`;
+const WEBSITE_URL = `https://yarnpkg.com`;
```

Comment:
Note: This line wasn't a problem before because currently I have to manually synchronize the sandbox changes - I made this change in the sandbox immediately after the playground PR was merged but I forgot to push it here too.